generators/server/templates/src/main/java/package/config/_LoadBalancedResourceDetails.java

```diff
@@ -0,0 +1,61 @@
+package <%=packageName%>.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.client.loadbalancer.LoadBalancerClient;
+import org.springframework.security.oauth2.client.token.grant.client.ClientCredentialsResourceDetails;
+import org.springframework.stereotype.Component;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+
+@Component
+public class LoadBalancedResourceDetails extends ClientCredentialsResourceDetails {
+    
+    Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
change to `private static final Logger log = LoggerFactory.getLogger(LoadBalancedResourceDetails.class);` for uniformity